modules/vaos/app/services/vaos/middleware/response/errors.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module VAOS
+  module Middleware
+    module Response
+      class Errors < Faraday::Response::Middleware
+        def on_complete(env)
+          return if env.success?
+
+          caller = "VAOS.#{caller_locations(14, 1)[0].label}"
```

Comment:
this gets the calling method e.g. VAOS.get_systems don't know if there's a better way to get the class and method that called it?

Reply:
This is kind of nice, but is it really necessary? I would think the backtrace should tell us what we need.